windows-installer/setup.nsi

```diff
@@ -36,6 +36,12 @@
 ; Directory where the fluentd config will be placed.
```

Comment:
Doesn't have to be this PR, but the lack of consistency amongst the directory vars should be fixed.
There are 3 formats right now:
- MAIN_**INSTDIR**
- FLUENTD_CONFIG_**DIRECTORY**
- CUSTOM_CONFIG_**DIR**

Reply:
Agreed. We might switch to a different packaging model before we get to it, though.